### PR TITLE
Adding manage exclusions tab to the module info

### PR DESCRIPTION
**Pull request by sypets**
Replacement for PR #111

### CODE REVIEW

**Comment by sypets** [comment]
@atigiti I am taking a look now.

File: Resources/Public/JavaScript/Brofix.js [hunk 1]
```diff
@@ -29,6 +29,33 @@ define(['jquery'], function($) {
     $('#titleButton').on('click', function (){
       $('#title_searchFilter').val('')
     })
+
+    $('#excludeUrlButton').on('click', function (){
+      $('#excludeUrl_filter').val('')
+    })
+
+    $('.selectAllLinks').click(function() {
+      alert('hello woeld')
```
Comment (sypets) [inline Resources/Public/JavaScript/Brofix.js]: Remove alert

**Comment by sypets** [comment]
@atigiti Thank you very much for your patch. I will merge now. I like the functionality very much.

There are still some issues, but they can be taken care of in a followup patch:

- permissions must be considered: if editor does not have access to `tx_brofix_exclude_link_target`, it should not be possible to remove the excluded URLs
- show hidden using icons as customary in TYPO3

**Comment by atigiti** [comment]
Thank you for taking this PR seriously, i tried to respect the low coupling between your code and what i added for this functionality, i think if you plan to add more tabs and functionnalities, it will be better if you can use some abstract classes for tab rendering,

**Comment by sypets** [comment]
@atigiti I noticed after merging when testing this in a site which already has several exclusions, there is a problem if you scroll down and then click on one of the checkboxes.

Tested with 182 exclusions, scroll down to the 12th one, select checkbox, then looks like this:
image


after click:

image


The viewport is reduced vertically, so only a small part (or nothing) of the list is visibly. I have not yet figured out what is causing this.

**Comment by sypets** [comment]
> i tried to respect the low coupling between your code and what i added for this functionality, i think if you plan to add more tabs and functionnalities, it will be better if you can use some abstract classes for tab rendering,

Feel free to add PR for that. It might make sense to split up changes to add functionality and to refactor for better class structure.

**Comment by sypets** [comment]
can only reproduce the scroll / resize above with chrome, not with firefox.

**Comment by sypets** [comment]
Is fixed with PR #120. I do not want to spend too much time there to investigate now. It is planned to move the module from the info module into a separate module and we may adapt some styles as well in the process.

**Pull request merged**